templates/reconciliation.js

```diff
@@ -42,7 +84,9 @@ var nextPairing = function nextPairing($currentPairing){
     highlightExistingVotes($nextPairing);
     $nextPairing.show();
   } else {
-    $('.messages').html('<h1>Reconciliation complete!</h1>'); 
+    $('.messages').html(
+      '<h1>Reconciliation complete!</h1>' + getDuplicateIDsHTML()
+    );
```

Comment:
It might be better to display this any time the CSV tray is opened, not only when you get to the very end. Often, when reconciling a lot of data, I'll do the reconciliation in multiple phases, and it would be great to be warned of duplicates at any time I 'finish' rather than only when I'm completely done.